lib.go

```diff
@@ -525,6 +526,15 @@ func (vm *VM) IBCPacketTimeout(
 	return &result, gasReport.UsedInternally, nil
 }
 
+func compileCosts(code WasmCode) uint64 {
+	// CostPerByte is how much CosmWasm gas is charged *per byte* for compiling WASM code.
+	// Benchmarks and numbers (in SDK Gas) were discussed in:
+	// https://github.com/CosmWasm/wasmd/pull/634#issuecomment-938056803
+	const CostPerByte uint64 = 3 * 140_000
+
+	return CostPerByte * uint64(len(code))
```

Comment:
Having this here feels great. That allows us to reprice based on our CosmWasm internal price target of 1 Terragas/second. But that's a separate step.